include/guards.hrl

```diff
@@ -1,13 +0,0 @@
-%%% -*-mode:erlang;coding:utf-8;tab-width:4;c-basic-offset:4;indent-tabs-mode:()-*-
```

Comment:
Not deleted, but moved and then changed: https://github.com/miniclip/gen_rpc/pull/4/files#diff-f8110bb2365868bb428d41358cb158bf6297756944dd57de101affa5be85eb1b.